src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -46,6 +46,7 @@ const useGroupInfo = (channelId = null) => {
   const {channel, profileChannel} = channelState;
   const createChat = channelState.channel?.data?.created_at;
   const anonUserEmojiName = channelState?.channel?.data?.anon_user_info_emoji_name;
+  const anonUserColorName = channelState?.channel?.data?.anon_user_info_color_name;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The introduction of the variable `anonUserColorName` is a good change as it will be used to generate the official anonymous username.
```diff
+ const anonUserColorName = channelState?.channel?.data?.anon_user_info_color_name;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->